src/pages/editor.js

```diff
@@ -6,7 +6,9 @@ import { useStaticQuery, graphql } from 'gatsby';
 import StoryblokClient from 'storyblok-js-client';
 import Components from '../components/components';
 
-const sbClient = new StoryblokClient({});
+const sbClient = new StoryblokClient({
+  accessToken: process.env.GATSBY_STORYBLOK_ACCESS_TOKEN,
```

Comment:
@sherakama @yvonnetangsu this seems to resolve the relations issue in the editor 😬

Reply:
Hmm or alternatively, we can store the preview key(s) in Vault so we can have one for PROD and DEV?

Reply:
eg: https://alumni.stanford.edu/editor?access_key={TOKEN GOES HERE}&path=/home